routes/web.php

```diff
@@ -13,6 +13,8 @@
 |
 */
 
-Route::get('/', function () {
-    return view('welcome');
-});
+Route::get('/', 'App\Http\Controllers\ConfigController@index')->name('index');
+Route::post('/createGame', 'App\Http\Controllers\ConfigController@createGame')->name('GoToGame');
+Route::get('/removeGame/{id}', 'App\Http\Controllers\ConfigController@removeGame')->name('RemoveGame');
+Route::get('/game/{id}', 'App\Http\Controllers\ConfigController@gameScreen')->name('GameScreen');
+Route::post('/storeKeys', 'App\Http\Controllers\ConfigController@storeKeys');
```

Comment:
Het werkt wel maar dit is de oude manier voor het definiëren van routes. De nieuwe manier zou netter zijn en ook consistenter in vergelijking met routes die al eerder gemaakt zijn.
![image](https://user-images.githubusercontent.com/27761262/112988347-087b7d80-9164-11eb-93ad-2ae6bc947bb9.png)